README.md

```diff
@@ -133,6 +133,7 @@ Default rules:
         "anon-function-style": "auto",
         "aa-comma-style": "no-dangling",
         "no-print": "off",
+        "no-stop": "warn",
```

Comment:
Should it be an error by default maybe? There's no good reason to keep a stop in the code, ever.

Reply:
I think warn is a good default. Some developers still use the `stop` statement while debugging. if bslint emits errors, that would prevent the transpile from happening and prevent sideloading possibly.

Reply:
Hmm well yes we don't want to prevent transpilation